app/main.py

```diff
@@ -2,0 +1,10 @@
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("cp command can take only 2 arguments")
+    cp_command, file, new_file = command.split()
```

Comment:
Code Style: The variable name `cp_command` is misleading. It suggests that the variable contains a command, but it actually contains a command name. A better name could be `command_name`.